extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml

```diff
@@ -29,8 +29,8 @@
         <version>1.0</version>
 
         <configuration>
-          <nodeVersion>v5.4.0</nodeVersion>
-          <npmVersion>3.3.12</npmVersion>
+          <nodeVersion>v8.1.2</nodeVersion>
+          <npmVersion>5.0.3</npmVersion>
```

Comment:
I haven't had a look at the sources yet, so thanks for bringing this up that early.
Yes, we do need approval for those too 😭